src/libs/deploy.js

```diff
@@ -245,12 +240,12 @@ export const deployAirWatch = async (signedApp, platform, awOrgID, awFileName, w
   const uploadOptions = {
     headers: {
       'Content-Type': 'application/octet-stream',
-      'aw-tenant-code': awTenantCode,
+      'aw-tenant-code': awKeyPairs[awKeys[2]],
```

Comment:
With these I'd consider naming the index so we can know what is in each slot of the array. For example if 0 was a private key and 1 a public key consider doing:
```
const publicKey = 0;
const privateKey = 1;
```

Then access via `awKeyPairs[awKeys[publicKey]]`

Reply:
will update the convention after getting device account